SampleGoGrpc/SimpleMath/grpc.go

```diff
@@ -0,0 +1,259 @@
+//Generated by GRPC Go plugin
+package SimpleMath
+
+import "github.com/google/flatbuffers/go"
+
+import (
+  grpc "google.golang.org/grpc"
+  context "golang.org/x/net/context"
+)
+
+// FlatCodec implements Grpc-go Codec
+
+type FlatCodec struct{}
+
+func (FlatCodec) Marshal(v interface{}) ([]byte, error) {
+  return v.(*flatbuffers.Builder).FinishedBytes(), nil
+}
+
+func (FlatCodec) Unmarshal(data []byte, v interface{}) error {
+  v.(flatbuffersInit).Init(data, flatbuffers.GetUOffsetT(data))
+  return nil
+}
+
+func (FlatCodec) String() string {
+  return "flatbuffers"}
+
+type flatbuffersInit interface {
+  Init(data []byte, i flatbuffers.UOffsetT)
+}
+
+// Client API for Math service
+type MathClient interface{
+  Mod2(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (* NumRes, error)  
+  AddNumbers(ctx context.Context, opts... grpc.CallOption) (Math_AddNumbersClient, error)  
+  Squares(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (Math_SquaresClient, error)  
+  IsOdd(ctx context.Context, opts... grpc.CallOption) (Math_IsOddClient, error)  
+}
+
+type mathClient struct{
+  cc *grpc.ClientConn
+}
+
+func NewMathClient(cc *grpc.ClientConn) MathClient {
+  return &mathClient{cc}
+}
+
+func (c *mathClient) Mod2(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (* NumRes, error){
+  out := new(NumRes)
```

Comment:
output is passed on to Invoke [method](https://github.com/JRonak/flatbuffers/blob/85ae154ac4ee351a4c8cd9db4baa9142236cf42e/SampleGoGrpc/SimpleMath/grpc.go#L49) of grpc and data is unmarshalled into out.
How could alloc be avoided in this case ?